app/domain/export/tabular/people/participation_nds_row.rb

```diff
@@ -29,6 +29,10 @@ def canton
       entry.canton.to_s.upcase
     end
 
+    def ahv_number
```

Comment:
Das brauchts vielleicht dann auch in anderen Youth dependant wagons und ihren speziellen exporten?

Reply:
@ThomasEllenberger wie ist das vorgehen hierfür? Hast du eine Liste mit den Wagons für mich, die ich mir ansehen soll?